Source/Utils/ScreenCover.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ScreenCover.swift
+//  walmart
+//
+//  Created by David Sica on 5/9/16.
+//  Copyright © 2016 Walmart. All rights reserved.
+//
+
+import Foundation
+
+// Declaring conformance to this Protocol will whitelist the UIViewController's
+//  view.
+//
+//  Optionally, implement the shouldWhitelist method to determine whether to
+//  wholesale whitelist at runtime.  Or, declare to NOT whitelist but manually 
+//  hide sensitive information on the view.
+@objc protocol ScreenCoverProtocol {
+    optional func shouldWhitelist() -> Bool
```

Comment:
It seems like this method is not really needed/used. You could just have an empty protocol that you use to "decorate" classes/interfaces that would declare the class is whitelisting itself from the screen covering.